Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -42,10 +42,29 @@ contains installation-wide configuration.
 :file:`config/sites/`
 ~~~~~~~~~~~~~~~~~~~~~
 
-The folder :file:`config/sites` contains
+The folder :file:`config/sites/` contains
 subfolders for each :ref:`site configuration <sitehandling>`.
 
 
+.. _directory-config-system:
+
+:file:`config/system/`
+~~~~~~~~~~~~~~~~~~~~~
+
+The folder :file:`config/system/` contains the
+:ref:`Configuration files <configuration-files>` :file:`config/system/settings.php`
+and :file:`config/system/additional.php`.
+
+
+..  versionchanged:: 12.0
+    For Composer-based installations the configuration files have been moved and
+    renamed:
+
+    *   :file:`public/typo3conf/LocalConfiguration.php` is now available in
+        :file:`config/system/settings.php`
+    *   :file:`public/typo3conf/AdditionalConfiguration.php` is now available
+        in :file:`config/system/additional.php`
+
```

Comment:
Please add

    This path can be retrieved from the Environment API, see :ref:`Environment-config-path`.

Like for legacy installation.

See also: https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/Core/Environment.php#L232